### ISSUE TITLE
Changelog entry for #8006

### ISSUE THREAD

**Issue by jneira** [haskell/cabal#8040]
* I missed #8006 lack of changelog ~~and regression test for #7279~~

**Comment by Mikolaj** [comment]
However, @gbaz says there "fair enough. this seems to trip some tests too. i'll take another look.", so it seems to be tested already.

**Comment by jneira** [comment]
but #7279 was a regression and we dont have a test to ensure it will not happen again, no?

**Comment by Mikolaj** [comment]
> but #7279 was a regression and we dont have a test to ensure it will not happen again, no?

@gbaz, @phadej: do we?

**Comment by jneira** [comment]
> > but #7279 was a regression and we dont have a test to ensure it will not happen again, no?
> 
> @gbaz, @phadej: do we?

yeah we do, sorry for the confusion, i forgot doctests, the test is here:

https://github.com/haskell/cabal/blob/4c9590e9e5a67dfe60cd94fe379d987aa09e05e1/Cabal-syntax/src/Distribution/Types/Flag.hs#L266-L273
